src/box/lua/console.c

```diff
@@ -1116,8 +1120,14 @@ lua_rl_getcompletion(lua_State *L)
 		 * metamethod instead of simply doing a raw lookup in the
 		 * metatable.
 		 */
+		lua_rawgeti(L, LUA_REGISTRYINDEX, gettable_ref);
+		lua_pushvalue(L, -3);
 		lua_pushstring(L, "__autocomplete");
-		lua_gettable(L, -3);
+		if (lua_pcall(L, 2, 1, 0) != 0) {
+			/* pcall returns an error to the stack */
+			lua_pushnil(L);
+			lua_replace(L, -2);
+		}
```

Comment:
I guess we'd better `return 0` as in case of `__autocomple` function error for consitency. We don't need to fallback to pass the test.

Reply:
Okay, let's do it this way.